cmd/buildah/addcopy.go

```diff
@@ -52,7 +67,16 @@ func addAndCopyCmd(c *cli.Context, extractLocalArchives bool) error {
 		return errors.Wrapf(err, "error reading build container %q", name)
 	}
 
-	err = builder.Add(dest, extractLocalArchives, args...)
+	options := buildah.AddAndCopyOptions{}
+	if chown := c.String("chown"); chown != "" {
```

Comment:
Why does this have to be two, I would think you should just do a 
if c.String("chown") != "" {
Then split it. I don't think we need to care of group is not set.

--chown 1234
Should be ok, and then just leave the other component alone.
}